packages/entity-browser/src/components/DetailView/DetailForm.js

```diff
@@ -28,9 +28,23 @@ export const DetailForm = props => {
           )
         }
       } else {
-        if (props.entity.paths[field.name].value !== null) {
-          const type = props.entity.paths[field.name].value.type
-          result.push(<Field name={field.name} type={type} key={i} label={field.label} component={LabeledField}/>)
+        const fieldProperties = props.entity.paths[field.name]
+        if (fieldProperties && fieldProperties.value != null) {
```

Comment:
refactor suggestion:

put this:

` result.push(<Field name={field.name} key={i} label={field.label}   component={LabeledField} />)`

outside of if/elseif and just determine the different props inside if and add with ...props/>